This is required since default icon is not configurable.
Also, if I simply use default.png as a impostor svg, it is being found out (errors in the webui)

diff --git a/internal/web/server.go b/internal/web/server.go
index f3d4850..0820a1c 100644
--- a/internal/web/server.go
+++ b/internal/web/server.go
@@ -605,9 +605,9 @@ func parseManifestEntries() {
 
 func defaultAvatar() string {
 	if dev {
-		return "http://localhost:16157/default.png"
+		return "http://localhost:16157/avatar.svg"
 	}
-	return config.C.ExternalUrl + "/" + manifestEntries["default.png"].File
+	return config.C.ExternalUrl + "/" + manifestEntries["avatar.svg"].File
 }
 
 func asset(file string) string {
diff --git a/public/main.ts b/public/main.ts
index e9934aa..bf56375 100644
--- a/public/main.ts
+++ b/public/main.ts
@@ -1,7 +1,7 @@
 import './style.scss';
 import './favicon-32.png';
 import './opengist.svg';
-import './default.png';
+import './avatar.svg';
 import dayjs from 'dayjs';
 import relativeTime from 'dayjs/plugin/relativeTime';
 import 'dayjs/locale/cs';
